beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -45,6 +45,12 @@ const (
 	eth1dataTimeout     = 2 * time.Second
 )
 
+// blindBlobsBundle holds the KZG commitments and other relevant sidecar data for a builder's beacon block.
+var blindBlobsBundle *enginev1.BlindedBlobsBundle
+
+// fullBlobsBundle holds the KZG commitments and other relevant sidecar data for a local beacon block.
+var fullBlobsBundle *enginev1.BlobsBundle
```

Comment:
will it be possible for fullBlobsBundle or blindBlobsBundle to be misaligned to their associated block somehow?

Reply:
This is a good point. To prevent leakover, I think we set the variable to nil after it's used.

Reply:
[ee64a3b](https://github.com/prysmaticlabs/prysm/pull/12956/commits/ee64a3b864d74e2cc1661ddede7fcf49f06d5375)